fiftyone/utils/eval/coco.py

```diff
@@ -489,20 +389,18 @@ def _coco_evaluation_iou_sweep(gts, preds, config):
         gts, preds, id_keys, iou_key, config, max_preds=config.max_preds
     )
 
-    matches_dict = {
-        i: _compute_matches(
+    return [
```

Comment:
Using floating point numbers as dict keys is dangerous as subtle numeric precision errors can cause lookups to fail. This hadn't arisen in the previous implementation *yet*, but better to avoid it anyway